app/services/feature_flag.rb

```diff
@@ -58,6 +58,11 @@ def feature
       :unlock_teachers_self_service_portal_account,
       "Unlock a teacher's self service portal account if their TRN is found",
       "Ransom Voke Anighoro"
+    ],
+    [
+      :ask_for_trn,
+      "Ask users for their TRN if they already know it",
+      "Ransom Voke Anighoro"
```

Comment:
This feature actually doesn't need a feature flag, because it's part of the identity work. We can use that feature flag instead.

Reply:
Do you mean `from_get_an_identity` ?

Reply:
Yes

Reply:
Removed